tools/fast/series/import_and_store.go

```diff
@@ -13,16 +14,23 @@ import (
 
 // ImportAndStore imports the `data` to the `client`, and proposes a storage
 // deal using the provided `ask`, returning the cid of the import and the
-// created deal.
+// created deal. It uses a duration of 10 blocks
 func ImportAndStore(ctx context.Context, client *fast.Filecoin, ask porcelain.Ask, data files.File) (cid.Cid, *storagedeal.Response, error) {
+	return ImportAndStoreWithDuration(ctx, client, ask, 10, data)
+}
+
+// ImportAndStoreWithDuration imports the `data` to the `client`, and proposes a storage
```

Comment:
This is to support the need in the deals show test to provide a duration so >1 payment voucher would be created.  I figured this would probably be useful so I added it here rather than copying almost all of the function and making it private to the test package.